### ISSUE TITLE
chinese zh.wttr.in web view error

### ISSUE THREAD

**Issue by GuanAllen** [chubin/wttr.in#119]
[<img width="1280" alt="screen shot 2017-06-03 at 12 47 29 am" src="https://cloud.githubusercontent.com/assets/14134195/26735930/a114d822-47f6-11e7-868a-9424d39aea9b.png">](zh.wttr.in)
[<img width="1280" alt="screen shot 2017-06-03 at 12 49 12 am" src="https://cloud.githubusercontent.com/assets/14134195/26735931/a126cc1c-47f6-11e7-9cb1-cea7a632c193.png">](zh.wttr.in)

**Comment by chubin** [comment]
And what about PNG?

zh.wttr.in/Taipei.png

Are there errors?

**Comment by GuanAllen** [comment]
The table is not aligned.

**Comment by chubin** [comment]
The PNG version seems to be ok (at least aligned, but some names seems to be displayed wrong), so the bug regard only the HTML version

**Comment by liyishuai** [comment]
> The PNG version seems to be ok (at least aligned, but some names seems to be displayed wrong)

I was to submit an issue on PNG rendering and found you're aware of it. Is it worth opening a new ticket?

**Comment by chubin** [comment]
@liyishuai What issue do you mean? That some characters (for evening and night) are missing? This one? Os something else? We need better support of different Chinese versions anyway (see #83); we should start from that issue, because the rest can be just a follow-up of it (e.g. because some wrong characters in the translation are used etc.).

**Comment by liyishuai** [comment]
I meant the Chinese characters are missing in PNG output. http://wttr.in/Taipei.png?lang=zh doesn't render Chinese characters correctly, but http://wttr.in/Taipei.png?lang=ja does. We probably need to configure the encoding for PNG generator.
I'll also contribute to language-related issues in the translation thread.

**Comment by chubin** [comment]
After #422 fixes rendering works almost properly, except one minor thing:
the date field is too narrow; space for one or two characters should be added

**Comment by playgithub** [comment]
same problem

https://wttr.in/bejin?lang=zh-cn
![image](https://github.com/user-attachments/assets/b1142c76-9dde-4962-8161-c115688b02ff)
